<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorSyntaxCheckWidget</class>
 <widget class="QWidget" name="EditorSyntaxCheckWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="grpEnableAutoSyntaxCheck">
     <property name="title">
      <string>Enable Auto Syntax Check</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chkSyntaxCheckWhenSave">
        <property name="text">
         <string>Check when save/load file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkSyntaxCheckWhenLineChanged">
        <property name="text">
         <string>Check when count of lines changed</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
